resources/views/inputs/includes/input-error-tooltip.blade.php

```diff
@@ -1,15 +1,18 @@
 @props([
     'error',
     'id',
+    'shifted' => false,
```

Comment:
@ItsANameToo basically I added `shifted` because it shifts from its original position.
But if you prefer I can change the argument name.

![image](https://user-images.githubusercontent.com/2118799/125420007-876f1b9f-8944-42a6-ba68-9fe67e321f22.png)

Reply:
ah I see 🤔 well I can't think of a super good name for that either :olegcornercry: let's leave it as is